### PR TITLE
WebUI: file download via webui #23

### PR DESCRIPTION
**Pull request by sprenner**

### CODE REVIEW

File: tools/bootstrap_tests.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=barisits/CN=692443/CN=Martin Barisits', 'x509', 'martin.barisits@cern.ch'),
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=tbeerman/CN=722011/CN=Thomas Beermann', 'x509', 'thomas.beermann@cern.ch'),
                                 ('/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=ruciobuildbot/CN=692443/CN=Robot: Rucio build bot', 'x509', 'rucio.build.bot@cern.ch'),
+                                ('CN=Stefan Prenner,CN=822876,CN=sprenner,OU=Users,OU=Organic Units,DC=cern,DC=ch', 'x509', 'stefan.prenner@cern.ch'),
```
Comment (mlassnig) [inline tools/bootstrap_tests.py:53]: this doesn't look right?

Comment (tbeerman) [reply tools/bootstrap_tests.py:53]: It works, at least on his dev machine. If ordered in the 'right' way the server does not accept the certificate.

File: lib/rucio/web/ui/static/did.js [hunk 2]
```diff
@@ -99,6 +100,7 @@ load_replicas = function(scope, name) {
     r.list_replicas({
         'scope': scope,
         'name': name,
+	'schemes': ['davs'],
```
Comment (tbeerman) [inline lib/rucio/web/ui/static/did.js]: I'm afraid you will have to do this call twice. If you do it like this only replicas which are on an RSE that supports davs are listed. But in the table it should always show all available replicas and then mark those that are on an RSE that is browser enabled.

**Comment by bari12** [comment]
Can you rebase this to a current master. Github shows that this is pulling in 250+ commits and 150k lines of code, so this seems a bit fishy 😄 It also shows conflicts. For the PR against next it seems to be correct though.
```
git fetch --all
git checkout patch-1630-file_download_webui
git rebase upstream/master
git push origin patch-1630-file_download_webui --force
```

**Comment by bari12** [comment]
Forget my comments, now with the repaired next it is fine. 😄

**Review by tbeerman — APPROVED** [review]

**Pull request merged**